benchmarks/src/main/java/io/grpc/benchmarks/driver/LoadClient.java

```diff
@@ -403,11 +403,13 @@ public void run() {
               @Override
               public void onNext(Messages.SimpleResponse value) {
                 delay(System.nanoTime() - now);
```

Comment:
onNext won't be called again. If it is, then an exception will be thrown because onCompleted is called more than once.